Assets/Drag.cs

```diff
@@ -35,11 +64,35 @@ private void StoppedMotion(ManipulationEventData arg0)
         {
             Destroy(gameObject); // delete the extra block
         }
-
+        else
+        {
+            transform.parent = console.transform;
+            gameObject.GetComponent<Drag>().enabled = false;
```

Comment:
you can remove the word `gameObject` here, similar to using the keyword `this` in c++ where it is implied/poor style to keep